sicario.c

```diff
@@ -28,6 +35,10 @@ bool userEscAction = false;
 bool userEnterAction = false;
 
 char gameBoard[BOARD_WIDTH][BOARD_HEIGHT];
+Player players[PLAYER_AMOUNT];
+int playerCount = PLAYER_AMOUNT;
+int decreaseGameBoardCount = 0;
+clock_t before;
```

Comment:
This should be a global variable because this variable measures the in game time. So even though it is only one function that handles it, it needs to survive beyond the scope of the function itself.

Reply:
Actually is not necessary to be a global variable, it can be a main scope variable, and the function that handles the time in the project, can receive this attribute as parameter.